### PR TITLE
map22 (skipped 40, 42, 55, 57)

### PR DESCRIPTION
**Pull request by carrotcake227**

### CODE REVIEW

File: Translation Script/Map107.csv [hunk 1]
```diff
@@ -10,59 +10,59 @@ so you'd rather not if possible..."
...
-見ちゃったわね♥"
+見ちゃったわね♥","\n<Lily>Sway sway♥Shake shake♥
+Aha♥
+You can't help but look, can you?♥"
 "\n<リリー>あんたのそのレベルじゃ
 こーんなあからさまな誘惑でも
-おちんちんにきちゃうでしょ♥","\n<リリー>あんたのそのレベルじゃ
-こーんなあからさまな誘惑でも
-おちんちんにきちゃうでしょ♥"
+おちんちんにきちゃうでしょ♥","\n<Lily>Those levels of yours...
```
Comment (OfficialLayton) [inline Translation Script/Map107.csv:63]: Ditto for this translation too ^^

File: Translation Script/Map107.csv [hunk 2]
```diff
@@ -110,50 +110,50 @@ one more day. If only...)"
...
+"リリーの日記",Lily's Diary
 "リリーの日記
 ミスト屋敷襲撃","Lily's Diary
 Misty Mansion Attack"
-"読んでみる","Read"
-"やめておく","Maybe not."
-"EV019","EV019"
-"ニンニクとお肉。","Meat and garlic."
-"\n<\n[1]>（料理をしている時間はないね。）","\n<\n[1]>（This isn't the time to be cooking.）"
+"読んでみる",Read
```
Comment (OfficialLayton) [inline Translation Script/Map107.csv:119]: Oh dear, we can't have this. If we remove the commas like this, the csv will break!

That wouldn't be good at all. Definitely go through and fix these if you can haha

File: Translation Script/Map107.csv [hunk 3]
```diff
@@ -110,50 +110,50 @@ one more day. If only...)"
...
+"やめておく",Maybe not.
+"EV019",EV019
+"ニンニクとお肉。",Meat and garlic.
+"\n<\n[1]>（料理をしている時間はないね。）",\n<\n[1]>（This isn't the time to be cooking.）
 "\n<\n[1]>（塩コショウとニンニクで焼くだけ。
 肉料理はこれで完成されている。あとは蛇足だ。）","\n<\n[1]>(Garlic, salt and pepper. Everything you need for a
 perfect meat dish. Anything else is just flashy nonsense.)"
-"調味料・・・","Spices..."
-"\n<\n[1]>（淫魔にコショウをかけてくしゃみをしている間に・・・）","\n<\n[1]>（I'd probably sneeze while I sprinkle pepper\non the succubi...）"
+"調味料・・・",Spices...
```
Comment (OfficialLayton) [inline Translation Script/Map107.csv:127]: Ditto commas for this.

File: Translation Script/Map107.csv [hunk 4]
```diff
@@ -110,50 +110,50 @@ one more day. If only...)"
...
+"\n<\n[1]>（淫魔にコショウをかけてくしゃみをしている間に・・・）",\n<\n[1]>（I'd probably sneeze while I sprinkle pepper\non the succubi...）
 "\n<\n[1]>（・・・
 これは持っていく必要はないかな。）","\n<\n[1]>（...
 I'm not going to take anything.）"
 "\n<\n[1]>（料理は苦手だ・・・
 少々とか適量とか、それが分からないからレシピを見ているのに。）","\n<\n[1]>(I'm no good at cooking...
 I never know if I'm using too much or too little,
 and I always end up looking back at the recipe.）"
-"麦・・・","Flour..."
```
Comment (OfficialLayton) [inline Translation Script/Map107.csv:136]: Commas.

File: Translation Script/Map107.csv [hunk 5]
```diff
@@ -110,50 +110,50 @@ one more day. If only...)"
...
-"\n<\n[1]>（そんなアホな戦い方はしたくない。）","\n<\n[1]>（I'd feel foolish fighting like that.）"
+"麦・・・",Flour...
+"\n<\n[1]>（焼きたてのパンが食べたい・・・）",\n<\n[1]>（I would love some fresh baked bread about now...）
+"お皿・・・",Plates...
+"\n<\n[1]>（投げて武器にすることができるかも・・・）",\n<\n[1]>（I might be able to use this as a throwing weapon...）
+"\n<\n[1]>（そんなアホな戦い方はしたくない。）",\n<\n[1]>（I'd feel foolish fighting like that.）
 "\n<\n[1]>（これをフリスビーのように投げれば・・・
 ・・・
-いや、やめておこう。）",""
-"アイテム","Items"
```
Comment (OfficialLayton) [inline Translation Script/Map107.csv:144]: Commas.

**Review by OfficialLayton — CHANGES_REQUESTED** [review]
Sorry for taking so long on the review Carrot! Thanks again for your help!
Just a slight translation change request and editing those commas back in would be great.

After those are done I'd be happy to approve. Thanks again ^^

**Comment by carrotcake227** [comment]
Sorry for the delay! School is picking up for me again and I was having trouble with git which kind of demotivated me to work on this haha. I'll try to get through more of it soon. 

Also, when you said commas, did you mean quotation marks? That's what I interpreted it as.

**Review by OfficialLayton — APPROVED** [review]

File: Translation Script/Map107.csv [hunk 6]
```diff
@@ -10,59 +10,59 @@ so you'd rather not if possible..."
...
 新鮮な生ごみ。","Looks like a failed recipe...
 These scraps are still fresh."
-"出現防止",""
-"EV011",""
-"リラクゼーションの本だ・・・","A book on relaxation..."
-"\n[1]は\C[3]『休憩』\C[0]を覚えた！","\n[1] learned \C[3]『Rest』\C[0]!"
-"イベント用キャラ","イベント用キャラ"
+"出現防止",
+"EV011",
+"リラクゼーションの本だ・・・",A book on relaxation...
```
Comment (OfficialLayton) [inline Translation Script/Map107.csv:30]: More missing quotation marks here too!

File: Translation Script/Map107.csv [hunk 7]
```diff
@@ -10,59 +10,59 @@ so you'd rather not if possible..."
 1斬　2打撃　3水　4火　5雷
 6誘惑　7食べ物
 特殊206（個別攻撃種類）
-1リンゴ　2皿　3卵",""
-"カチコチに凍っている・・・","It's frozen solid..."
-"食糧庫ドア","Pantry Door"
-"食堂の鍵を開けた！","Unlocked the door with the Dining Room Key!"
+1リンゴ　2皿　3卵",
+"カチコチに凍っている・・・",It's frozen solid...
```
Comment (OfficialLayton) [inline Translation Script/Map107.csv:14]: Here as well!

**Review by OfficialLayton — CHANGES_REQUESTED** [review]

**Comment by OfficialLayton** [comment]
Hello @carrotcake227! 

I am very, very sorry for the late reply, I've been busy myself so I can totally understand. Wishing you all the best for your school stuffs! Once again, thank you very, very much for your help. I can't thank you enough! And please, feel free to go at your own pace.  If you're having any more trouble with git in the future let me know and I'd be happy to help. It's the least I can do ^^
Ack, you're totally right. I did mean the quotation marks! Basically without them the csvs will be read incorrectly.

For example: 
`"カチコチに凍っている・・・",It's frozen solid...`
Should be:
`"カチコチに凍っている・・・","It's frozen solid..."`

I did still seem some more cases of missing quotation marks, did you manage to get them all?

**Comment by OfficialLayton** [comment]
Looks good to me!

Sorry about the lateness in getting this merged. Hopefully nothing breaks and thank you very, very much Carrot! Much appreciated ^^

**Comment by OfficialLayton** [comment]
Ah, sorry my bad @carrotcake227 it appears there are some merge conflicts, let me know once they're done and I'll merge after that ^^

**Comment by OfficialLayton** [comment]
Heyo @carrotcake227!

Sorry about the late reply, I know you're busy, so I had a look at the merge conflicts and they *seem* to be just from the comma error that happened recently. I've done my best to resolve them myself so you don't have to worry about it ^^

With this, I should be able to just merge what you've done into the translation! Thanks again for your help :D

**Pull request merged**